src/Example/Controllers/HomeController.cs

```diff
@@ -15,7 +15,10 @@ public ActionResult Index()
         {
             var user = GetUser();
 
-            ViewBag.Photo = "data:image/jpeg;base64," + Convert.ToBase64String(user.Photo);
+            if (user.Photo == null)
+                ViewBag.Message = "No photo provided, change the application settings to request a photo from the user for this demo";
+            else
+                ViewBag.Photo = "data:image/jpeg;base64," + Convert.ToBase64String(user.Photo);
```

Comment:
Shouldn't this happen in the SDK?

Reply:
This code is just choosing what to do with the user profile information retrieved. It is assuming that the user is always sharing their photo. This is fine if this is always specified by the application, but this is just to alert anyone as to why the photo isn't there, as before it was just crashing.

Reply:
As discussed, we'll look at doing the conversion to base64 string in the SDK, so this conversion above doesn't have to be done by every consumer of the API